### ISSUE TITLE
Expose EventPipe Tracing API

### ISSUE THREAD

**Issue by brianrob** [dotnet/coreclr#15375]
Currently, there isn't a supported way to control the EventPipe from in-proc.

**Comment by yuleyule66** [comment]
Will the EventPipe Tracing API be released in 2.1 ?

**Comment by brianrob** [comment]
@yuleyule66, no this is not going to be released in 2.1.

**Comment by liuhaoyang** [comment]
I'm very interested in EventPipe. Where can I see the latest developments in EventPipe API?

**Comment by jzabroski** [comment]
Just follow Brian Robbins github activity feed: https://github.com/brianrob
- @brianrob

**Comment by noahfalk** [comment]
@brianrob - what sort of control did you have in mind? My understanding is that EventListener is capable of controlling all the events and levels of any EventSource. Presumably what can't be controlled is asking EventPipe to stream the data somewhere else, such as to a file. Is that the type of control you were thinking of?

**Comment by brianrob** [comment]
@noahfalk, yes, that's right.  If you're using EventListener then you're good to go, but if you want to stream or send it to a file, there's no nice way to do that.  There are only private APIs in System.Private.CoreLib.
